gdpr/anonymizers/fields.py

```diff
@@ -326,7 +326,7 @@ def get_replacement_file(self):
     def get_encrypted_value(self, value: FieldFile, encryption_key: str):
         file_name = value.name
         value.delete(save=False)
-        file = self.get_replacement_file()
+        file = self.get_replacement_file(file_name)
```

Comment:
I could also send the value and pull the name out of it in the function itself, but then I would be doing that twice?